### DISCUSSION TITLE
Adding vite-specific dependencies just for the renderer?

### DISCUSSION THREAD

**Discussion by Lexpeartha** [cawa-93/vite-electron-builder#646] [Q&A] [answered]
Let's say I need to add vite plugin/dependency just for the renderer process - for example [auto importing of vue components](https://github.com/antfu/unplugin-vue-components); what am I supposed to do?
I tried adding few plugins renderer process, and managed to configure them (or at least I thought I did) and they worked like they were supposed to on dev server, but after running build command, executable that I got just starts with an empty browser window.

I guess this could also extend to adding maybe some css framework you will use only in renderer process, but I think perhaps there should be something in `README.md` that shows how to add process specific dependency so it doesn't break the rest of the project

**Comment by cawa-93** [comment, marked as answer]
It should work. In some of my projects, I installed dependencies and plugins (such as [windi-css](https://github.com/windicss/vite-plugin-windicss)) and used them only in renderer without issues. I assume that the problem is elsewhere.

**Comment by Lexpeartha** [comment]
So you added them `vite.config.ts` that is in `renderer` folder?
That might be the issue, because I just put all my plugins in root config file without much thinking
